bundles/binding/org.openhab.binding.mios/src/main/java/org/openhab/binding/mios/internal/config/ServiceAliases.properties

```diff
@@ -109,3 +109,4 @@ Tuning1=urn:micasaverde-com:serviceId:Tuning1
 VideoAdjustment1=urn:micasaverde-com:serviceId:VideoAdjustment1
 Volume1=urn:micasaverde-com:serviceId:Volume1
 WMC1=urn:micasaverde-com:serviceId:WMC1
+RGBController1=urn:upnp-org:serviceId:RGBController1
```

Comment:
Why is this in the upnp-org domain?  That's reserved strictly for use by standards emitted by the standards body.  I took a quick look, but didn't see that one.